### ISSUE TITLE
Unable to resolve action `skywall/universal-apk-builder@v1`, unable to find version `v1`

### ISSUE THREAD

**Issue by rovkinmax** [skywall/universal-apk-builder#7]
How I use this action
```
      - name: Build Universal APK
        uses: skywall/universal-apk-builder@v1
        with:
          aab_path: "app/build/outputs/bundle/prodRelease/app-prod-release.aab"
          keystore_path: "/app/release.jks"
          keystore_password: ${{secrets.STOREPASS}}
          keystore_alias: ${{secrets.KEYALIAS}}
          keystore_alias_password: ${{secrets.KEYPASS}}  
          output_dir: app/build/outputs/apk/prod/release/
```

**Comment by rovkinmax** [comment]
fix with `skywall/universal-apk-builder@v1.0.1` I think than need update example

**Comment by skywall** [comment]
@rovkinmax Fixed. Thank you.
